collision_clearance: 0.10

planning_groups:
  -
    name: L_ARM
    joints:
      - L_SFE
      - L_SAA
      - L_HR
      - L_EB
      - L_WR
      - L_WFE
      - L_WAA
    end_effector: L_PALM
  -
    name: R_ARM
    joints:
      - R_SFE
      - R_SAA
      - R_HR
      - R_EB
      - R_WR
      - R_WFE
      - R_WAA
    end_effector: R_PALM

collision_links:
  - L_UPPER_ARM
  - L_FOREARM
  - L_HAND_L_FINGER_KNUCKLE_2
  - L_HAND_L_FINGER_KNUCKLE_3
  - L_HAND_R_FINGER_KNUCKLE_2
  - L_HAND_R_FINGER_KNUCKLE_3
  - L_HAND_M_FINGER_KNUCKLE_2
  - L_HAND_M_FINGER_KNUCKLE_3
  - L_LOWER_WRIST
  - R_UPPER_ARM
  - R_FOREARM
  - R_HAND_L_FINGER_KNUCKLE_2
  - R_HAND_L_FINGER_KNUCKLE_3
  - R_HAND_R_FINGER_KNUCKLE_2
  - R_HAND_R_FINGER_KNUCKLE_3
  - R_HAND_M_FINGER_KNUCKLE_2
  - R_HAND_M_FINGER_KNUCKLE_3
  - R_LOWER_WRIST

joint_costs:
  L_SFE: 1.0
  L_SAA: 1.0
  L_HR: 1.0
  L_EB: 1.0
  L_WR: 1.0
  L_WFE: 1.0
  L_WAA: 1.0
  R_SFE: 1.0
  R_SAA: 1.0
  R_HR: 1.0
  R_EB: 1.0
  R_WR: 1.0
  R_WFE: 1.0
  R_WAA: 1.0

joint_velocity_limits:
  L_SFE: 2.0
  L_SAA: 2.0
  L_HR: 2.0
  L_EB: 2.0
  L_WR: 3.0
  L_WFE: 3.0
  L_WAA: 3.0
  R_SFE: 2.0
  R_SAA: 2.0
  R_HR: 2.0
  R_EB: 2.0
  R_WR: 3.0
  R_WFE: 3.0
  R_WAA: 3.0

reference_frame: BASE
num_feature_basis_functions: 1

collision_space:
  size_x: 2.6
  size_y: 1.3
  size_z: 2.0
  origin_x: -1.3
  origin_y: 0.0
  origin_z: 0.0
  resolution: 0.02
  field_bias_x: 0.0
  field_bias_y: 0.0
  field_bias_z: 0.0
